terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/FunctionTestSupport.java

```diff
@@ -16,6 +16,7 @@
 package org.terasoluna.gfw.functionaltest.app;
 
 import java.io.File;
+import java.time.Duration;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
```

Comment:
Unnecessary import statements remain.

Reply:
40bc931b4a8e961bbca35c58395e7176a7340595
deleted the unnecessary import statement .